modules/aggregation-layers/src/aggregation-layer-v9/aggregator.ts

```diff
@@ -0,0 +1,65 @@
+import type {Attribute, BinaryAttribute} from '@deck.gl/core';
+
+export type AggregationOperation = 'SUM' | 'MEAN' | 'MIN' | 'MAX';
```

Comment:
Worth adding `'COUNT'`? For the GPU aggregation case it would avoid uploading a buffer containing just `1s`